src/views/CharacterBuild.js

```diff
@@ -11,86 +11,78 @@ const useStyles = makeStyles({
     },
     "& .Mui-focused": {
       color: "#90caf9", //Color of label when focused
-    }
-  }
+    },
+  },
 });
 
-export default function CharacterBuild() {
+function CharacterBuild(props) {
   const classes = useStyles();
-  // a character build that a user will give us
-  const build = {
-    character_name: "Chongyun",
-    attack_base: 733,
-    attack_total: 1675,
-    DMG_Bonus_Cryo: 46,
-    crit_damage: 194.6,
-  };
-
-  //cleans up inputted build (above)
-  const charBuild = new CharacterClass(build);
-  const baseDamage = CharacterBaseDamage(charBuild);
 
   return (
-    <>
-      <Box style={{ display: "flex", alignItems: "center" }}>
-        <Box
-          style={{
-            justifyContent: "flex-start",
-            minWidth: "250px",
-            textAlign: "right"
-          }}
-          marginRight="20px"
-        >
-          character_name
-        </Box>
-        <Box style={{ justifyContent: "flex-end", minWidth: "100px" }}>
-          {get(charBuild, "character_name")}
-        </Box>
-      </Box>
-      {map(keys(omit(charBuild, "character_name")), (stats, idx) => {
-        return (
-          <Box key={idx} style={{ display: "flex", alignItems: "center" }}>
-            <Box
-              style={{
-                justifyContent: "flex-start",
-                minWidth: "250px",
-                textAlign: "right"
-              }}
-              marginRight="20px"
-            >
-              {stats}
-            </Box>
-            <Box style={{ justifyContent: "flex-end", minWidth: "100px" }}>
-              <TextField
-                type="number"
-                label={stats}
-                inputProps={{
-                  style: {
-                    color: "white",
-                    fontFamily: "genshinFont"
-                  }
-                }}
-                size="small"
-                className={classes.field}
-                onChange={e => (charBuild[stats] = parseFloat(e.target.value))}
-                defaultValue={get(charBuild, stats)}
-              ></TextField>
-            </Box>
-          </Box>
-        );
-      })}
-      {map(keys(baseDamage), (talent, idx) => {
+    <Box>
+      {map(STAT_FIELDS, (stats, idx) => {
         return (
-          <Box key={idx} style={{ display: "flex" }}>
-            <Box style={{ justifyContent: "flex-start", minWidth: "250px" }}>
-              {talent}
-            </Box>
-            <Box style={{ justifyContent: "flex-end", minWidth: "100px" }}>
-              {get(get(baseDamage, talent), "damage")}
-            </Box>
+          <Box key={`${stats}${idx}`}>
```

Comment:
string / number for key so easier to see

Reply:
![image](https://user-images.githubusercontent.com/55843764/174419012-6faac141-a69d-4c76-ae79-461a83ae84a7.png)